app/src/main/java/org/p2p/wallet/history/model/HistoryItem.kt

```diff
@@ -1,9 +1,17 @@
 package org.p2p.wallet.history.model
 
+import org.p2p.wallet.moonpay.model.MoonpaySellTransaction
+import org.p2p.wallet.sell.ui.lock.SellTransactionDetails
 import org.threeten.bp.ZonedDateTime
 
 sealed class HistoryItem {
     data class TransactionItem(val transaction: HistoryTransaction) : HistoryItem()
     object Empty : HistoryItem()
     data class DateItem(val date: ZonedDateTime) : HistoryItem()
+
+    data class MoonpayTransactionItem(val transactionDetails: SellTransactionDetails) : HistoryItem() {
```

Comment:
Looks like we need mapper fto make SellTransactionDetails.toMoonpayTransactionItem